dot/network/status.go

```diff
@@ -61,28 +61,28 @@ func (status *status) setHostMessage(msg Message) {
 // handleConn starts status processes upon connection
 func (status *status) handleConn(conn network.Conn) {
 	ctx := context.Background()
-	peer := conn.RemotePeer()
+	thispeer := conn.RemotePeer()
```

Comment:
is this necessary? I would prefer if we didn't change this but maybe I am misunderstanding something herre?

Reply:
yes that is required, we cant have a variable that shadows the import with the same name.

Reply:
maybe `remotePeer`? not a huge fan of `this`

Reply:
sure